src/main/java/finalcountdown/homerecipesystembackend/controller/PrepMethodController.java

```diff
@@ -0,0 +1,73 @@
+package finalcountdown.homerecipesystembackend.controller;
+
+import finalcountdown.homerecipesystembackend.model.*;
+import finalcountdown.homerecipesystembackend.service.PrepMethodService;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.net.URI;
+import java.util.List;
+
+@RestController
+@Slf4j
+@RequestMapping("/prep-method")
+public class PrepMethodController {
+    private final PrepMethodService prepMethodService;
+
+    public PrepMethodController(PrepMethodService prepMethodService) {
+        this.prepMethodService = prepMethodService;
+    }
+
+    @PostMapping("/create")
+    public ResponseEntity<?> createPrepMethod(@RequestBody PrepMethod newPrepMethod) {
```

Comment:
We cannot create a PrepMethod object, because it has a list of steps.
We need to use DTO-s like we use in RecipeController and IngredientController create endpoints